### DISCUSSION TITLE
How to specify the file open permissions as readonly from workspace file

### DISCUSSION THREAD

**Discussion by aghayanlena** [eclipse-theia/theia#13976] [General]
Hi, 

Is there a way to specify the file permission for the files within the workspace? Does Thea have a configuration similar to the "settings.files.readonlyInclude" available in Visual Studio Code?

Thanks, 
Lena

**Comment by msujew** [comment]
Hey @aghayanlena,

I don't think this setting is currently supported. Do you mind creating an issue for that? In case you're interested in contributing the feature, please feel free to open a PR :)
